tools/bigquery-zos-mainframe-connector/README.md

```diff
@@ -1,326 +1,172 @@
 This example has 3 steps:
-1. Upload COBOL dataset as ORC to a bucket
-2. Load ORC file to BigQuery table
-3. Query BigQuery table
+* Upload COBOL dataset as ORC to a bucket
+* Load ORC file to BigQuery table
+* Query BigQuery table
 
 ```
 //STEP01 EXEC BQSH
 //INFILE DD DSN=<HLQ>.DATA.FILENAME,DISP=SHR
 //COPYBOOK DD DISP=SHR,DSN=<HLQ>.COPYBOOK.FILENAME
 //STDIN DD *
-gsutil cp --replace gs://bucket/tablename.orc
+BUCKET=my-long-bucket-name-1234
+gsutil cp --replace gs://$BUCKET/tablename.orc
 /*
 //COND02 IF (STEP01.RC = 0) THEN
 //STEP02 EXEC BQSH
 //STDIN DD *
-bq load --project_id=myproject \
+PROJECT=my-long-project-name-1234
+bq load --project_id=$PROJECT \
   myproject:DATASET.TABLE \
   gs://bucket/tablename.orc/*
 /*
 //COND03 IF (STEP02.RC = 0) THEN
 //STEP03 EXEC BQSH
-//QUERY DD DSN=<HLQ>.QUERY.FILENAME,DISP=SHR
+//QUERY DD DSN=<HLQ>.QUERY.FILENAME,DISP=SHR 
 //STDIN DD *
-bq query --project_id=myproject \
+PROJECT=my-long-project-name-1234
+bq query --project_id=$PROJECT \
   myproject:DATASET.TABLE \
   gs://bucket/tablename.orc/*
 /*
 ```
 
+### Enabling Debug Logging
+Debug logging can be enabled by editing the BQSH Procedure and setting the environment variable BQSH_ROOT_LOGGER=DEBUG \
+Debug logging can be disabled by commenting out the line or setting the variable to anything other than DEBUG.
 
-### Subcommand Usage
-
-#### gsutil
+## Using with Google Cloud Storage
 
-```
-gsutil (gszutil-1.0.0)
-Usage: gsutil [cp|rm] [options] destinationUri
-
-  --help                   prints this usage text
-Command: cp [options]
-Upload Binary MVS Dataset to GCS
-  --replace                delete before uploading
-  --partSizeMB <value>     target part size in megabytes (default: 256)
-  --batchSize <value>      blocks per batch (default: 1000)
-  -p, --parallelism <value>
-                           number of concurrent writers (default: 6)
-  --timeOutMinutes <value>
-                           timeout in minutes (default: 60)
-Command: rm [options]
-Delete objects in GCS
-  -r, --recursive          delete directory
-  -f, --force              delete without use interaction (always true)
-  destinationUri           Destination URI (gs://bucket/path)
-  --dataset_id <value>     The default dataset to use for requests. This flag is ignored when not applicable. You can set the value to [PROJECT_ID]:[DATASET] or [DATASET]. If [PROJECT_ID] is missing, the default project is used. You can override this setting by specifying the --project_id flag. The default value is ''.
-  --location <value>       A string corresponding to your region or multi-region location.
-  --project_id <value>     The project ID to use for requests. The default value is ''.
-  --allow_non_ascii        allow non ascii characters
-  --stats_table <value>    tablespec of table to insert stats
-  --max_error_pct <value>  job failure threshold for row decoding errors (default: 0.0
-```
+Check the [Using with Google Cloud Storage](./USING_WITH_GCS) guide.
 
```

Comment:
the link doesn't seem to work when I view it on github

Reply:
it should be working now